src/lib/Scenes/MyProfile/MyProfileEditForm.tsx

```diff
@@ -0,0 +1,305 @@
+import { useNavigation } from "@react-navigation/native"
+import { EditableLocation } from "__generated__/ConfirmBidUpdateUserMutation.graphql"
+import { MyProfileEditForm_me$key } from "__generated__/MyProfileEditForm_me.graphql"
+import { MyProfileEditFormQuery } from "__generated__/MyProfileEditFormQuery.graphql"
+import { useFormik } from "formik"
+import { Image } from "lib/Components/Bidding/Elements/Image"
+import {
+  buildLocationDisplay,
+  DetailedLocationAutocomplete,
+} from "lib/Components/DetailedLocationAutocomplete"
+import { FancyModalHeader } from "lib/Components/FancyModal/FancyModalHeader"
+import LoadingModal from "lib/Components/Modals/LoadingModal"
+import { useFeatureFlag } from "lib/store/GlobalStore"
+import { getConvertedImageUrlFromS3 } from "lib/utils/getConvertedImageUrlFromS3"
+import { showPhotoActionSheet } from "lib/utils/requestPhotos"
+import { compact, isArray } from "lodash"
+import { Avatar, Box, Button, Flex, Input, Join, Spacer, Text, Touchable, useColor } from "palette"
+import React, { Suspense, useContext, useRef, useState } from "react"
+import { ScrollView, TextInput } from "react-native"
+import { useFragment, useLazyLoadQuery } from "react-relay"
+import { graphql } from "relay-runtime"
+import * as Yup from "yup"
+import { updateMyUserProfile } from "../MyAccount/updateMyUserProfile"
+import { MyProfileContext } from "./MyProfileProvider"
+
+const PRIMARY_LOCATION_OFFSET = 240
+
+export interface EditMyProfileValuesSchema {
+  photo: string
+  name: string
+  displayLocation: { display: string | null }
+  location: EditableLocation | null
+  profession: string
+  otherRelevantPositions: string
+  bio: string
+}
+
+export const editMyProfileSchema = Yup.object().shape({
+  photo: Yup.string(),
+  name: Yup.string().required("Name is required"),
+  bio: Yup.string(),
+})
+
+export const MyProfileEditForm: React.FC<{ me: MyProfileEditForm_me$key }> = (props) => {
+  const me = useFragment<MyProfileEditForm_me$key>(meFragment, props.me)
+
+  const color = useColor()
+  const navigation = useNavigation()
+  const scrollViewRef = useRef<ScrollView>(null)
+  const { showActionSheetWithOptions } = useActionSheet()
+
+  const nameInputRef = useRef<Input>(null)
+  const bioInputRef = useRef<TextInput>(null)
+
+  const [loading, setLoading] = useState<boolean>(false)
+  const [didUpdatePhoto, setDidUpdatePhoto] = useState(false)
+
+  const enableCollectorProfile = useFeatureFlag("AREnableCollectorProfile")
+
+  const { localImage, setLocalImage } = useContext(MyProfileContext)
+
+  const uploadProfilePhoto = async (photo: string) => {
+    try {
+      // We want to show the local image initially for better UX since Gemini takes a while to process
+      setLocalImage(photo)
+      const iconUrl = await getConvertedImageUrlFromS3(photo)
+      await updateMyUserProfile({ iconUrl })
+    } catch (error) {
+      console.error("Failed to upload profile picture ", error)
```

Comment:
The arguments will be joined with a whitespace by `console.log`.
```suggestion
      console.error("Failed to upload profile picture", error)
```

Reply:
Isn't that what we want, we need whitespace to make it readable right?

Reply:
`console.log("first", "second")` will print `first second`.

Reply:
🤔 💭